ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -110,7 +111,18 @@ public void onPageStarted(WebView webView, String url, Bitmap favicon) {
               SystemClock.nanoTime(),
               createWebViewEvent(webView, url)));
     }
-
+    @Override
+    public boolean shouldOverrideUrlLoading(WebView view, String url){
+        ReactContext reactContext = (ReactContext) view.getContext();
+        if(url.startsWith("tel:")){
```

Comment:
I think we shouldn't do anything special only for `tel:`. Probably a better way is to check if it's not `http:` or `https:` and handle the intent.